docs/app/pages/components/components-sections/tree-view/tree-grid/snippets/app.html

```diff
@@ -9,7 +9,7 @@
             <thead>
                 <tr class="treegrid-header table-header-dark">
                     <th>
-                        <ux-checkbox aria-label="Select all rows" [(value)]="selectAll"></ux-checkbox>
+                        <ux-checkbox class="m-r-sm" aria-label="Select all rows" [(value)]="selectAll"></ux-checkbox>
```

Comment:
Could you remove the bootstrap class and update the library stylesheet to ensure proper spacing for existing and future implementations of the tree grid?